src/library/scala/collection/immutable/Seq.scala

```diff
@@ -78,7 +78,7 @@ trait IndexedSeq[+A] extends Seq[A]
 }
 
 object IndexedSeqDefaults {
-  def defaultApplyPreferredMaxLength: Int = sys.props.getOrElse(
+  val defaultApplyPreferredMaxLength: Int = sys.props.getOrElse(
```

Comment:
Let's use `System.getProperty` here to avoid the risk of introducing an initialization cycle.

Reply:
@retronym updated to use System.getProperty